test/draw/gpu/GpuSurfaceTest.ooc

```diff
@@ -98,9 +98,10 @@ GpuSurfaceTest: class extends Fixture {
 			DrawState new(gpuImage) setInputImage(this sourceImage) setDestinationNormalized(quadrantTopLeft) setTransformNormalized(FloatTransform3D createRotationX(180.0f toRadians())) draw()
 			DrawState new(gpuImage) setInputImage(this sourceImage) setDestinationNormalized(quadrantBottomRight) setTransformNormalized(FloatTransform3D createRotationX(180.0f toRadians())) draw()
 			rasterFromGpu := gpuImage toRaster()
-			expect(rasterFromGpu distance(correctImage), is equal to(0.0f) within(0.05f))
+			expect(rasterFromGpu distance(correctImage), is less than(0.05f))
 			(rasterFromGpu, correctImage, gpuImage) free()
 		})
+		/* Lines are thinner on some GPUs
 		this add("draw shapes", func {
```

Comment:
A new test should replace it.
https://github.com/magic-lang/ooc-kean/issues/1892